### PR TITLE
[doc] Improve Pulsar Administration Pulsar proxy

### PR DESCRIPTION
**Pull request by Monica-zy**
Motivation
Improve the language and the overall descriptive style of the Pulsar Administration document (Administration-proxy section): http://pulsar.apache.org/docs/en/administration-proxy/

Modifications
Adjust the tone, personal pronouns, voice also some typo errors of some sentences in the document.

### CODE REVIEW

**Comment by Monica-zy** [comment]
@Anonymitaet Could you please help me review this, thanks~

File: site2/docs/administration-proxy.md [hunk 1]
```diff
@@ -4,27 +4,27 @@ title: The Pulsar proxy
...
+> If you use service discovery, the network ACL must allow the proxy to talk to the ZooKeeper nodes on the zookeeper client port, which is usually 2181, and on the configuration store client port, which is 2184 by default. Opening the network ACLs means that if someone compromises a proxy, they have full access to ZooKeeper. For this reason, using broker URLs to configure the proxy is more secure.
 
-### Option 2: Using broker URLs
+### Option 2: Use broker URLs
 
 The more secure method of configuring the proxy is to specify a URL to connect to the brokers.
 
-> [Authorization](security-authorization#enabling-authorization-and-assigning-superusers) at the proxy requires access to ZooKeeper, so if you are using this broker URLs to connect to the brokers, Proxy level authorization should be disabled. Brokers will still authorize requests after the proxy forwards them.
+> [Authorization](security-authorization#enable-authorization-and-assign-superusers) at the proxy requires access to ZooKeeper, so if you use this broker URLs to connect to the brokers, you should diable the Proxy level authorization. Brokers still authorize requests after the proxy forwards them.
```
Comment (Anonymitaet) [inline site2/docs/administration-proxy.md]:
```suggestion
> [Authorization](security-authorization#enable-authorization-and-assign-superusers) at the proxy requires access to ZooKeeper, so if you use these broker URLs to connect to the brokers, you should disable the Proxy level authorization. Brokers still authorize requests after the proxy forwards them.
```

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests

**Comment by Monica-zy** [comment]
run integration tests

**Pull request merged**